packages/genesys-spark-components/src/components/stable/gux-popover-list/gux-popover-list.tsx

```diff
@@ -51,12 +51,6 @@ export class GuxPopoverList {
   @Prop()
   position: Placement = 'bottom';
 
-  /**
-   * Indicate if the dismiss button is displayed
-   */
-  @Prop()
-  displayDismissButton: boolean;
```

Comment:
I don't see this as a breaking api change as the component should not have this. Also its removal will not break anything as there are other ways to close the popover-list component.

Reply:
This change seems like it should be in a separate PR from a dependency bump PR

Reply:
Ya fairpoint. Removed from this PR.